AssetDependencyGraph/Editor/AssetDependencyGraph.cs

```diff
@@ -87,6 +88,25 @@ public void OnEnable()
         {
             text = "Clear"
         });
+        
+        var ts = new ToolbarSearchField();
+        ts.RegisterValueChangedCallback(x => {
+            if (string.IsNullOrEmpty(x.newValue)) {
+                m_GraphView.FrameAll();
+                return;
+            }
+            
+            m_GraphView.ClearSelection();
+            // m_GraphView.graphElements.ForEach(y => { // BROKEN, Case 1268337
+            m_GraphView.graphElements.ToList().ForEach(y => {
+                if (y is Node node && y.title.Contains(x.newValue)) {
```

Comment:
```suggestion
                if (y is Node node && y.title.IndexOf(x.newValue, StringComparison.OrdinalIgnoreCase) >= 0) {
```
Not sure if it'd be worth us making this case insensitive, perhaps it's better to be specific when you're dealing with a large graph?

Reply:
I think let's go with insensitive for now; at some future point Searcher package integration would be cool but not for this PR :)